src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -191,7 +192,7 @@ namespace rewrite {
 	   (k == kind::FLOATINGPOINT_MIN_TOTAL) || (k == kind::FLOATINGPOINT_MAX_TOTAL));
 #endif
     if (node[0] == node[1]) {
-      return RewriteResponse(REWRITE_DONE, node[0]);
+      return RewriteResponse(REWRITE_AGAIN_FULL, node[0]);
```

Comment:
`REWRITE_AGAIN` should be sufficient here (to make sure that we rewrite as much as possible on the prerewriting pass).